napari/_vispy/layers/image.py

```diff
@@ -65,7 +65,7 @@ def get_node(
         return res
 
 
-class VispyImageLayer(VispyBaseLayer[_ImageBase]):
+class VispyVolumeBaseLayer(VispyBaseLayer[_ImageBase]):
```

Comment:
Why did you call it volume? To differentiate from Image? I find Volume confusing since this is for both 2D and 3D. Maybe "VispyGridLayer"? "VispyFieldLayer"? "VispyScalarFieldLayer"? (I know RGB breaks "scalar" a bit but I think it's still ok.)

Reply:
Yes. I prefer to not use image in base name for better readability. But I do not have idea for proper name. But this is private class so we could change name later. I will change to scalar.

Reply:
@jni `VispyNDarrayBaseLayer`?

Reply:
I don't mind that but I slightly prefer ScalarField. I would merge either option. 😊

Reply:
changed